src/txn/txn.c

```diff
@@ -1262,9 +1262,6 @@ __wt_txn_rollback(WT_SESSION_IMPL *session, const char *cfg[])
 
     /* Rollback and free updates. */
     for (i = 0, op = txn->mod; i < txn->mod_count; i++, op++) {
-        /* Assert it's not an update to the lookaside file. */
-        WT_ASSERT(session, S2C(session)->cache->las_fileid == 0 || !WT_IS_LAS(op->btree));
```

Comment:
Could you open a ticket to add a statistic that captures when a transaction on the history store is rolled back? That sounds like a useful thing to be able to track when debugging.

Reply:
Cerated WT-5435